client.go

```diff
@@ -379,11 +379,13 @@ func (c *Client) sendWithWriter(writer io.Writer, packet []byte) error {
 
 // Loop: Receive data from server
 func (c *Client) recv(keepaliveQuit chan<- struct{}) {
+	defer func() {
```

Comment:
nit: this could just be "defer close(keepaliveQuit)", the closure is unnecessary (though also unlikely to hurt).

Reply:
good point ! Thanks